tests/acceptance/features/bootstrap/SpacesTUSContext.php

```diff
@@ -166,4 +167,113 @@ public function userUploadsAFileWithContentToViaTusInsideOfTheSpaceUsingTheWebda
 		$this->spacesContext->setSpaceIDByName($user, $spaceName);
 		$this->tusContext->userUploadsAFileWithContentToUsingTus($user, $content, $resource);
 	}
+
+	/**
+	 * @When /^user "([^"]*)" uploads a file "([^"]*)" to "([^"]*)" with mtime "([^"]*)" via TUS inside of the space "([^"]*)" using the WebDAV API$/
+	 *
+	 * @param string $user
+	 * @param string $source
+	 * @param string $destination
+	 * @param string $mtime Time in human readable format is taken as input which is converted into milliseconds that is used by API
+	 * @param string $spaceName
+	 *
+	 * @return void
+	 *
+	 * @throws Exception
+	 * @throws GuzzleException
+	 */
+	public function userUploadsAFileToWithMtimeViaTusInsideOfTheSpaceUsingTheWebdavApi(string $user, string $source, string $destination, string $mtime, string $spaceName): void {
```

Comment:
too long lines. good to break them
```suggestion
	public function userUploadsAFileToWithMtimeViaTusInsideOfTheSpaceUsingTheWebdavApi(
	        string $user, 
	        string $source,
	        string $destination,
	        string $mtime,
	        string $spaceName
	): void {
```